### PR TITLE
Add support for braille terminal

### LINKED ISSUE #307: add Braille terminal support
hello it will help me a lot if quickemu can add Braille display

references: 
* https://gitlab.archlinux.org/archlinux/archiso/-/blob/master/scripts/run_archiso.sh#L82
* https://lists.nongnu.org/archive/html/qemu-devel/2008-04/msg00052.html
* search braille in qemu manual

### PR DESCRIPTION
**Pull request by philclifford**
fixes #307

Added parameter `--acc | --accessible | --accessibility` which if given `braille` ( or `brl ` )
with add parameters to invoke `Qemu` braille terminal support.

### CODE REVIEW

**Comment by alex19EP** [comment]
it doesn't work at the moment, but I think the problem is in qemu and not in the script. I'll try to figure it out.

**Comment by philclifford** [comment]
> it doesn't work at the moment, but I think the problem is in qemu and not in the script. I'll try to figure it out.

Thanks - for me it seemed to write what I was expecting into the calling script, and qemu seemed to try and start the vm but failed because there was no braille display/device (in what seemed a sensible and correct way).  Is there any output you can share that might give a clue. Если логи на русском, это не проблема. Я немного читаю по русски. Плохо и очень медленно ...

I really hope you can figure it out and we can get this working.

**Comment by alex19EP** [comment]
> Is there any output you can share that might give a clue.

i created [this](https://brltty.app/pipermail/brltty/2022-January/019079.html) thread on brltty's ml.

TLDR: all is working fign with sdl display but not with gtk one. don't know why yet.
so I think this can be merged.

p.s. мне всегда приятно видеть людей которые знают русский язык.

**Comment by philclifford** [comment]
> TLDR: all is working fign with sdl display but not with gtk one. don't know why yet. so I think this can be merged.
Excellent news - we can always add tweaks (or extra documentation) later if there's a way to support other displays.  I'll add a note to the documentation

> p.s. мне всегда приятно видеть людей которые знают русский язык.
Я понимаю - и мне всегда приятно познакомиться с новым русским другом.
Thanks again for the request, and your help.

**Comment by flexiondotorg** [comment]
Love this feature! Is there any reason why we should not always enable the braille character device(s) rather than have an option?

**Comment by philclifford** [comment]
@flexiondotorg braille display feature breaks all but SDL display - until qemu is fixed it needs fencing off (see above)
